CONTRIBUTING.MD

```diff
@@ -0,0 +1,56 @@
+# Instructions for contributing treebank annotations
+
+This file documents the procedure for contributing trees to the legal-cgel treebank. As of writing (08/12/2024), we are in the process of adding and adjudicating 'one-off' trees of (pseudo) randomly-selected sentences from the U.S. Code. These trees are maintained in `/datasets/oneoff/` (henceforth the 'oneoff' folder), with associated LaTeX source code and .pdf-rendered LaTeX in `/datasets/oneoff/tex` and `/datasets/oneoff/pdf`, respectively. 
+
+These instructions assume that you've been assigned one-off sentences to annotate, and that a project contributor with write privileges to `main` has added "stubs" of your annotations to the oneoff folder. A stub is a .cgel file with populated fields `# sent_id`, `# text`, `# sent`, and `# tree_by`. As of right now, assume one stub per tree. Here's an example stub: 
+
+```
+# sent_id = `60-3`
```

Comment:
I wouldn't put backticks in the sentence id

Reply:
thanks for catching this! will update.